RequestsPlus/__init__.py

```diff
@@ -1,2 +1,2 @@
-from RequestsPlus import *
-from ResponseCollection import *
+from . RequestsPlus import *
+from . ResponseCollection import *
```

Comment:
A comment here can be useful why python3 imports are different from python2. not needed though!